ocfweb/docs/docs/services/hpc.md

```diff
@@ -92,22 +96,29 @@ Linux on your system, you can install Singularity from the official apt repos:
 sudo apt install singularity-container
 ```
 
-If you are running Mac you can look [here][mac_install], or Windows [here][win_install].
+If you do not have an apt-based Linux distribution, installation instructions
+can be found [here][linux_install]. Otherwise, if you are running Mac you can
+look [here][mac_install], or Windows [here][win_install].
 
 #### Building Your Container
 
 ```
-singularity build --sandbox ./my_container docker://ubuntu
+singularity build [--sandbox] ./my_container docker://ubuntu
 ```
 This will create a Singularity container named `my_container`. If you are still
 testing out your container on your system, we suggest you use the `--sandbox`
 option. This will allow you to install new packages in your container as you
-need them. The `docker://ubuntu` option notifies Singularity to bootstrap the
-container from the official Ubuntu docker container on [Docker Hub][docker_hub].
-There is also a [Singularity Hub][singularity_hub], from which you can directly
-pull Singularity images in a similar fashon. We also have some pre-built
-containers that you may use to avoid having to build your own. They are
-currently located at `/home/containers` on the Slurm master node.
+need them. However, if you are working on our infrastructure you will *not*
+be able to install non-pip packages on your container, because you do not have
+`sudo` privileges. If you would like to create your own container with new packages,
+you must create the container on your own machine, using the above command with
+`sudo` prepended, and then transfer it over to our infrastructure. The
```

Comment:
Now they're gonna ask how to do that.

Reply:
For the transferring to our infrastrucure part?

Reply:
Yeah, I was thinking of that.